src/classes/CW_AdminControllerTest.cls

```diff
@@ -0,0 +1,199 @@
+@isTest
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.